buildlib/tools/builds.sh

```diff
@@ -348,6 +348,26 @@ build_cmake_examples() {
 	fi
 }
 
+#
+# Build with FUSE
+#
+build_fuse() {
+	if az_module_load $FUSE3_MODULE
+	then
+		echo "==== Build with FUSE (dynamic link) ===="
+		${WORKSPACE}/contrib/configure-devel --prefix=$ucx_inst --with-fuse3
+		$MAKEP
+
+		echo "==== Build with FUSE (static link) ===="
```

Comment:
pls clean all work directory including auto* output before doing the 2nd step (static link)